src/components/rooms/room.routes.ts

```diff
@@ -3,10 +3,33 @@ import { Request, Response, Router } from 'express'
 import { getBusyRooms, getEventsForRoom } from './room.service'
 import { ROOMS } from '../../util/constants'
 import { asyncWrapper } from '../../util/asyncWrapper'
+import { User } from '../users/user'
 
 export const index = asyncWrapper(async (req: Request, res: Response) => {
   const busyRooms = await getBusyRooms()
-  res.render('room/index', { busyRooms, ROOMS })
+  let rooms = ROOMS.map(floor => {
+    return {
+      floor,
+      busy: busyRooms.find(el => el.id == floor)
+    }
+  })
+  let userRoom
+  if (req.user) {
+    const user = (req.user as User)
+    const favorites = await User.relatedQuery('favorites').for(user.id)
+    if (favorites && favorites.length > 0)
+      rooms = rooms.map(room => {
+        return {
+          ...room,
+          favorite: favorites.some(el => el.room == room.floor)
+        }
+      }) 
+    if (user.floor && user.floor >= 3 && user.floor <= 18) {
```

Comment:
I think we can completely remove the `floor` from the use model as the favorites extends this functionality.

Reply:
Should I do it in this PR, or should I introduce a new issue for it.

Reply:
Do it in here since this feature makes the current functionality obsolete.